docs/how-does-version-resolution-work-in-maven-and-gradle.md

```diff
@@ -0,0 +1,117 @@
+# How does version resolution work in Maven and Gradle?
+
+The build system that you choose for building your Java code determines which
+version resolution algorithm is used to choose the versions of your
+dependencies. Unfortunately, the two most popular build systems in open source
+(Maven and Gradle) use different version resolution algorithms. They also use
+different names for the concept: Maven calls the process *dependency mediation*,
+while Gradle calls the process *version conflict resolution*. In this article,
+we will use the term *version resolution*. No matter which you choose, you are
```

Comment:
Version conflict resolution is fine after this note. It's less jargony than "dependency mediation." but let's not introduce a thrid synonym.

Reply:
There's a problem though - it overloads the "conflict" word. In our world (the JLBP project), we generally apply the word "conflict" to situations where something breaks, and we treat version resolution as a neutral process that doesn't have any "conflicts" if the result works. Using the word "conflict" for this part of the process in our documentation will muddy the waters. 

You could also view this differently: "version resolution" is just a simplified version of the term "version conflict resolution", and isn't really a distinct term. 

I will add some additional verbiage about omitting the word "conflict".